### ISSUE TITLE
DPs for energy consumptions are not selectable

### ISSUE THREAD

**Issue by codingcatgirl** [rospogrigio/localtuya#688]
## The problem

Now that the lack of updates on the energy confumption DPs was fixed, i decided to give loyaltuya another go.

Sadly, when setting up, i now can't even select the energy consumption DPs. When i add one of my smart plugs with energy monitoring with todays new update, only two DPs show up. The ones that would contain the energy consumption information are missing. When i tried localtuya for the first time a few months ago, a lot more DPS would show up (and provide data as long as the app was running).

## Environment

- Localtuya version: 3.3.0
- Home Assistant Core version: 2021.12.7
- [] Are you using the Home Assistant Tuya Cloud component ? 
- [] Are you using the Tuya App in parallel ? 

## Steps to reproduce
1. Click "Add Integration"
2. Setup a tuya smart plug with energy consumption feature.
3. The DPs for energy consumption can't be selected.

## Configuration `configuration.yaml` or `config_flow`
![image](https://user-images.githubusercontent.com/3868507/147885964-2bf1eb00-a485-48b9-b80c-4b3d3956daa3.png)

## DP dump
```
TuyaDebug (Tuya DPs dump) [1.0.0]

Device bf08d3a7087ecab2c7uxpp at 192.168.178.99 key xxxxxxxxxx protocol 3.3 dev_type type_0a:
    DPS [1] VALUE [True] 
    DPS [9] VALUE [0] 
```

## Provide Home Assistant taceback/logs

I'd like to set the log level to debug, but the method described in the issue template does not work for me. There seems to be no service named logger.set_level.

## Additional information
I tried to set the DPs using the YAML configuration and discovered another bug that prevented me from doing so: #687

**Comment by vmartinv** [comment]
Did you have DPs for this initially and they disappeared or you have never seen them? Are you sure your plugs actually have the DPs?

**Comment by codingcatgirl** [comment]
> Did you have DPs for this initially and they disappeared or you have never seen them? 

Like i said: "When i tried localtuya for the first time a few months ago, a lot more DPS would show up (and provide data as long as the app was running)."

> Are you sure your plugs actually have the DPs?

Well, since i got data from them previously while the app was open – yes.

> Maybe try adding the plug with the app open after rebooting home assistant, that way it can detect the dps being reported

I'm aware that this might be a workaround, but the entire point of #549 included in the last update was that you don't need the app to be open and this should not be needed to get localtuya working. Also, i have to assume that even with this workaround it might reset at some point again.

**Comment by vmartinv** [comment]
> I'm aware that this might be a workaround, but the entire point of #549 included in the last update was that you don't need the app to be open and this should not be needed to get localtuya working. Also, i have to assume that even with this workaround it might reset at some point again.

can you test if the workaround works first ? We need to be able to detect the DPs

**Comment by codingcatgirl** [comment]
I can, but it will be a while until i have the time to tinker around with the app again

**Comment by codingcatgirl** [comment]
@Elendilon just explained [here](https://github.com/rospogrigio/localtuya/issues/687#issuecomment-1003956928) that it is a zombie issue they have fixed with #491 – so once that PR is merged, the issue should hopefully disappear.

**Comment by codingcatgirl** [comment]
This has been fixed for me since the last update, though i had to remove all localtuya devices and set them up anew. But now it works! Including Power Consumption! Awesome! Thanks!
